src/edges_analysis/analysis/beams.py

```diff
@@ -260,6 +260,101 @@ def from_feko(cls, path: [str, Path], az_antenna_axis: float = 0) -> Beam:
             raw_file=path,
         )
 
+    @classmethod
+    def from_feko_raw(
+        cls,
+        file_name_prefix: [str, Path],
+        ext: str = "txt",
+        f_low: int = 40,
+        f_high: int = 100,
+        freq_p: int = 61,
+        theta_p: float = 181,
+        phi_p: float = 361,
+        az_antenna_axis: float = 0,
+    ) -> Beam:
+        """
+        Read a FEKO beam file.
+
+        Parameters
+        ----------
+        filename
+            The path to the file.
+        az_antenna_axis
+            The azimuth of the primary antenna axis, in degrees.
+
```

Comment:
Add all parameters to the docstring -- especially not sure what the `_p` parameters are?

Reply:
Note that the docstring has a certain format so that it can be written to html and hosted on readthedocs. The format for a parameter is

```
name
    Description
```